src/index.html

```diff
@@ -1,13 +1,375 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakerlab</title>
+    <link
+      rel="icon"
+      type="favicon"
+      href="./images/favicon.ico">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <!-- #region header + menu -->
+    <header class="header" id="header">
+      <div class="header__content">
+        <div class="top-bar page__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+              src="./images/logo.png"
+              alt="Bakerlab logo">
```

Comment:
```suggestion
              alt="Bakerlab logo"
            >
```
Consider fixing indentations accordingly to the code style guide